content/en/tracing/trace_collection/trace_context_propagation/dotnet.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Propagating .NET Trace Context
+kind: documentation
+code_lang: dotnet
+type: multi-code-lang
+code_lang_weight: 80
+further_reading:
+    - link: 'tracing/glossary/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
```

Comment:
Just noting that we have further reading links in the front matter, but we don't have the Further Reading section at the bottom of the article.